code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -65,6 +65,12 @@
 	pass_airlock_spell = new
 	AddSpell(pass_airlock_spell)
 
+/mob/living/simple_animal/hostile/morph/Destroy()
+	. = ..()
```

Comment:
actually, shouldn't this be after the QDEL_NULLs?

Reply:
Doesn't matter, either way works. I can just change it to a ``return ..()`` though

Reply:
bam